<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AffirmForm</class>
 <widget class="QWidget" name="AffirmForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AffirmForm
{
border-image: url(:/resource/images/common/Transparent_Black_70.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>271</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>271</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widgetBg" native="true">
     <property name="minimumSize">
      <size>
       <width>1025</width>
       <height>115</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widgetBg
{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>127</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widgetFunctionArea" native="true">
        <property name="minimumSize">
         <size>
          <width>765</width>
          <height>115</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>765</width>
          <height>115</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QLabel" name="labelTitle">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 25pt &quot;Tahoma&quot;;
color: rgb(82, 164, 225);</string>
         </property>
         <property name="text">
          <string>You Want?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelBack">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>70</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel
{
	font: 17pt &quot;Tahoma&quot;;
	color: rgb(0, 0, 0);
}
QLabel:hover
{
	font: 17pt &quot;Tahoma&quot;;
	color: rgb(82, 164, 225);
}
QLabel:pressed
{
	font: 17pt &quot;Tahoma&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelNotSave">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>70</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel
{
	font: 17pt &quot;Tahoma&quot;;
	color: rgb(0, 0, 0);
}
QLabel:hover
{
	font: 17pt &quot;Tahoma&quot;;
	color: rgb(82, 164, 225);
}
QLabel:pressed
{
	font: 17pt &quot;Tahoma&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Don't Save</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelSave">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>50</y>
           <width>161</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel
{
	font: 37pt &quot;Tahoma&quot;;
	color: rgb(0, 0, 0);
}
QLabel:hover
{
	font: 37pt &quot;Tahoma&quot;;
	color: rgb(82, 164, 225);
}
QLabel:pressed
{
	font: 37pt &quot;Tahoma&quot;;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>127</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
